### ISSUE TITLE
Request to update the website

### ISSUE THREAD

**Issue by Lucas-C** [py-pdf/py-pdf.github.io#5]
Hi @MartinThoma 🙂

Could the static website be updated please, following the merge of this PR in August?
https://github.com/py-pdf/py-pdf.github.io/pull/4

It seems that I do not have the necessary rights to manually run the `pages-build-deployment` workflow:
https://github.com/py-pdf/py-pdf.github.io/actions/workflows/pages/pages-build-deployment

**Comment by MartinThoma** [comment]
Yes, I see `fpdf2` now on https://py-pdf.github.io/ . I'm sorry for that failure :see_no_evil:

**Comment by Lucas-C** [comment]
> Yes, I see fpdf2 now on [py-pdf.github.io](https://py-pdf.github.io/) . I'm sorry for that failure 🙈

No worries, and thank you!

> **Permission check**: I want you to have those permissions. I don't know what is missing for that.

I wonder if it's not a matter of **Base permissions**: https://github.com/organizations/py-pdf/settings/member_privileges

We could either raise this to **Admin** or add me explicitely as a member to this repository, with higher permissions.

**Comment by MartinThoma** [comment]
I've added you to this repository as an admin as this seems the obvious no-regret choice (I'm uncertain about the other option and what else this might change).
